recipes/darts/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This file should already be in the tarball from GitHub? In that case, you shouldn't include the file in the recipe.

Reply:
weirdly it does not include the in the tarball.

Reply:
That is bizarre, but ok.